tools/wptrunner/wptrunner/browsers/base.py

```diff
@@ -275,3 +280,117 @@ def __call__(self, line):
         self.logger.process_output(self.pid,
                                    line.decode("utf8", "replace"),
                                    command=" ".join(self.command) if self.command else "")
+
+
+class WebDriverBrowser(Browser):
```

Comment:
Given that this PR has sat for so long now, is there code in here that was copied from other code now removed, and are there other changes in the meantime that need to be reflected here too?

Reply:
I looked and I didn't notice anything I missed, at least.